gulpfile.js

```diff
@@ -29,7 +30,17 @@ elixir(function(mix) {
     		'public/css/app.css'
     	], {
     		liveCSS: true
-    	});
+    	})
+        .karma({
+            jsDir:[
+                'public/js/vendor.js',
+                'node_modules/angular-mocks/angular-mocks.js',
+                'public/js/partials.js',
+                'public/js/app.js',
+                'tests/angular/**/*spec.js'
```

Comment:
@kujtimiihoxha Yes you're right
I think it's a great idea
I also love the `ng` generator idea.. I'll add a new issue for it